src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java

```diff
@@ -0,0 +1,22 @@
+package team.five.amazonreviewutil.controller;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import team.five.amazonreviewutil.service.ServiceParsing;
+
+@Component
+public class DataInsertController {
+
+    private static final String DATA_FILE_NAME = "D:\\My\\MA\\project\\Reviews.csv";
```

Comment:
@TatianaPirch what if I will run this on my Mac? Will it works?

Reply:
It's will not on work in another Windows(like mine or Kolya's) too;
Idk, i tried to see reliable path file in idea, but it don't worked for me;
@TatianaPirch please place .csv file in package src/main/resources/ and push it in branch too;

Reply:
String txt = "./src/resources/Reviews.csv" , try it out, maybe will work for you